### DISCUSSION TITLE
Inquiry About Using a Nickname for GitHub Verification

### DISCUSSION THREAD

**Discussion by AndroidBaaaby** [community/community#148144] [GitHub Education] [answered]
### Hi everyone,

I am in the process of submitting my academic affiliation for verification through GitHub Education. However, I got the rejection and it said:

"You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname."

I would prefer to use a nickname or a variation of my name in my GitHub profile instead of my full legal name. Would it be possible to retain a nickname in my GitHub profile while still completing the academic verification process? 

Best regards,
Abby

**Comment by aayush-ojha** [comment, marked as answer]
Hi @AndroidBaaaby ,

It's understandable that you'd prefer to use a nickname or a variation of your name on your GitHub profile,but unfortunately, for the academic verification process through GitHub Education, they require that your full legal name matches exactly with what is listed in your academic affiliation document. This is to ensure accuracy and consistency in the verification process.

Although once your verification has been completed, you could potentially update your name, but during the verification process it's important to use your full legal name.

**Comment by httperror505** [comment]
Given that you're trying to verify through academical documents, it is a must to use your full name to match your uploaded documents. Tho it is understandable that you protect your anonymity, so what you can do is revert your name into your desired nickname when verification process is over.
